tests/unit/services/seeder-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Service | seeder', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
+  test('it exists', function(assert) {
+    let service = this.owner.lookup('service:seeder');
+    assert.ok(service);
```

Comment:
I can fix up our existing tests for the services if you want to merge this first.

Reply:
Yep, let's merge this and add tests to service in a separated PR.